### ISSUE TITLE
Issues with Building - missing argument global setup

### ISSUE THREAD

**Issue by jonluntzel** [JaneliaSciComp/CohenBolstadLee_eLife2017#1]
Hello,

I am running into issues with step 7 (image attached). This error occurs even after waiting and/or rerunning the build command so it does not seem to be the kind that "causes the build script to exit with an error, even though it compiled."

![Screenshot (102)](https://user-images.githubusercontent.com/34121312/120704426-6af64000-c46b-11eb-9341-8c5f336cc471.png)

After receiving the error, I manually installed boost 1.58.0. The installation went further, but eventually ran into the error again. I used the solution found here to address it: [](https://stackoverflow.com/questions/56814269/boost-installation-missing-argument-global-setup) but continue to get the same error. It seemed like this solution was on the right track though.

I then started installation on a separate 64-bit machine for a sanity check and steps 1-6 run fine but do get the same error in step 7. I have not tried externally installing boost on the second machine just for a clean slate in case it causes more problems. Any advice?

edit: Tried it on a d drive and renamed the folder 'Jovian' to mirror the layout described in the readme; still get the error

**Comment by renderdude** [comment]
The one thing that immediately jumps out from the attached image, is that you’re using visual studio 2019.

We’ve only tested the build using VS 2017. He might try and install with that version of the compiler and see if that fixes the problem.

Mark

**Comment by jonluntzel** [comment]
VS 2017 does not receive the same error, so it may be fixing something. I tried through VS 2017 on 2 different machines and get the following error for one machine:
![Screenshot (6)](https://user-images.githubusercontent.com/34121312/121755971-75e54c00-cacd-11eb-9851-291d1ae1c528.png)
I can get past the Boost.Build step on the windows command line by installing (bootstrap, b2, add path variables) but msys does not respect those path variables regardless so it doesn't seem to matter if I can get boost installed outside of the mingw32 prompt. I hoped that setting the path variables within the mingw command prompt with export PATH=$PATH:.../include/boost-1_58 and .../lib would cause it to recognize an existing boost installation and skip the step that errors out when building mouseover. But it does not unfortunately.

**Comment by jonluntzel** [comment]
I may have encountered and fixed the recursion error on the second machine, but if so, do not remember how.. The second machine receives this error: 
![Screenshot (104)](https://user-images.githubusercontent.com/34121312/121757173-839cd080-cad1-11eb-801e-d7d1dee348d5.png)
I mostly care about the first machine but if this information helps at all that's good. In the readme it says 'Visual Studio 2013', I could try using VS 2013 instead?

**Comment by renderdude** [comment]
It’s been a long-time since I looked or even built that particular version of the software. I’m pretty sure I built with 17 before, but if the readme says 13, then that’s the correct version. 

The big show stopper is that the software *requires* windows 7 (I can’t tell which version of windows you’re running). It will not build on any later windows os. This is due to some of the third-party libraries (I’m looking at you Collada) didn’t support any newer version of windows. Getting a win7 machine today can be challenging. 

Hope that helps. 

Mark
